authors/apps/articles/urls.py

```diff
@@ -23,8 +21,13 @@
     path('articles/<slug:slug>/dislike/',
          LikesView.as_view(model=Articles, vote_type=LikeDislike.DISLIKE),
          name='article_dislike'),
-    path('articles/ratings/<int:pk>/', RetrieveUpdateDeleteRatingAPIView.as_view(), name='rating-detail'),
     path('articles/favorites/me/', GetUserFavoritesView.as_view(), name="get_favorites"),
     path('articles/<slug:slug>/favorite/', FavoriteView.as_view(), name="favorite"),
+    path('articles/reports/get',
```

Comment:
Is there a specific reason why you chose to have the url as `articles/reports/get` ? I'd recommend you use `articles/reports` if the purpose is just listing all reports that have been made just to keep the endpoints consistent

Reply:
I think the better way to go around is: user `reports/articles/` as a URL

Reply:
done and done,..